runtime/compiler/env/j9methodServer.cpp

```diff
@@ -536,6 +536,30 @@ TR_ResolvedJ9JITaaSServerMethod::fieldAttributes(TR::Compilation * comp, I_32 cp
    return result;
    }
 
+static bool doResolveAtRuntime(J9Method *method, I_32 cpIndex, TR::Compilation *comp)
+   {
+   TR_J9VMBase *fej9 = (TR_J9VMBase *)(comp->fe());
+   auto stream = fej9->_compInfoPT->getMethodBeingCompiled()->_stream;
+   if (!method)
```

Comment:
I would put this test at the very top for a fast return in case it's a frequent case.
Also, please add brackets everywhere to follow our coding conventions. I know that the original code doesn't have them, but people will look at this code when we have to merge it to master.

Reply:
You mean put `doResolveAtRuntime` before the remote call?
I don't think it's possible, because this method takes `J9Method *` as an argument, which is fetched from the client.
I will add brackets.

Reply:
I meant to move
```
if (!method)
   return true;
```
at the very top and avoid getting the `fej9` and `stream` when method is NULL

Reply:
Done.